AudioBar/AudioBarViewController.swift

```diff
@@ -96,6 +108,43 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
         }
     }
 
+    private func configureCommandCenterCommands() {
+        commandCenter.playCommand.addTarget { [weak store] _ in
```

Comment:
Do we really need `.remoreCommand` message? Can't we go with `store!.dispatch(.userDidTapPlayButton)`?